digitalstrategie/urls.py

```diff
@@ -17,23 +18,27 @@
     path('admin/', include(wagtailadmin_urls)),
     path('documents/', include(wagtaildocs_urls)),
     path('i18n/', include('django.conf.urls.i18n')),
-    re_path(r'^sitemap\.xml$', wagtail_sitemap),
     re_path(r'^robots\.txt$', TemplateView.as_view(
         template_name='robots.txt',
         content_type="text/plain"), name="robots_file"),
-    path('search/', SearchResultsView.as_view(), name="search"),
     path('newsletter-signup/', newsletter_view, name="newsletter-signup"),
```

Comment:
I somehow could not make it work for the newsletter signup, but also could not test properly.. do you think this is important to also have the language there, so I should try again @fuzzylogic2000?

Reply:
Hmm, we are not giving the language to the newsletter provider, right? That was in kosmo...
So, I don't think, we need the language here, but maybe we should talk to @maxliqd again.